sql/insert.go

```diff
@@ -83,6 +83,10 @@ func (p *planner) Insert(n *parser.Insert, autoCommit bool) (planNode, *roachpb.
 			addIfDefault(*col)
 		}
 	}
+	// DEFAULT VALUES do count default columns as inputs.
```

Comment:
instead of this, can you increment `numInputColumns` in `addIfDefault`? This style is quite a bit harder to decypher, despite generating a smaller diff.